src/hotspot/share/runtime/flags/jvmFlagConstraintsCompiler.cpp

```diff
@@ -302,6 +302,9 @@ JVMFlag::Error TypeProfileLevelConstraintFunc(uintx value, bool verbose) {
 
 JVMFlag::Error InitArrayShortSizeConstraintFunc(intx value, bool verbose) {
   if (value % BytesPerLong != 0) {
+    JVMFlag::printError(verbose,
+                       "InitArrayShortSize (" INTX_FORMAT ") must be "
+                        "a multiple of %d\n", value, BytesPerLong);
```

Comment:
Consider indenting them a bit. I think the line starting from `"a...` has one excess whitespace.

Reply:
Good catch. Fixed. Thanks for the review!